### ISSUE TITLE
Alt på logotyp i sidfoten + alt i länktext

### ISSUE THREAD

**Issue by peterk** [Kungbib/frontend-guide#19]
http://kungbib.github.io/frontend-guide/section-3.html#section-3.3

``` html
<img src="./assets/kb_logo_black.svg" class="logo-medium" />
```

Om den inte har någon aktivitet knuten till sig kan vi använda alt=""?

I samma fragment finns ett alt-attribut på a-elementet. Kan fimpas.

**Comment by klngwll** [comment]
FIxed by 887d7da617ac707d35787aaf454a66114e3d51f1
